.github/workflows/benchmark_execution.yml

```diff
@@ -76,6 +76,7 @@ jobs:
         run: |
           ./build_tools/benchmarks/export_benchmark_config.py \
             execution \
+            --ci \
```

Comment:
Hmm I don't love this. Won't this mean that the configs from the CI won't be directly usable? What about instead adding an arg to the benchmark tool that says which target to run for? Someone should probably know which of those they are trying to run. Could also make the benchmark tool *only* take things thus wrapped with a command-line argument to specify which to run. If there's exactly one of them it could run that one (would require attempting to parse and then failing with a helpful message if unsuccessful).

Reply:
A `--target_device_name` arg sounds good to me. Added.